### ISSUE TITLE
LoggerMessageDefineAnalyzer throws OutOfRangeException when parameter is empty

### ISSUE THREAD

**Issue by pranavkm** [dotnet/roslyn-analyzers#5362]
### Analyzer

LoggerMessageDefineAnalyzer

### Analyzer source

**SDK**:  6.0.100-rc.1.21410.3

### Describe the bug

Reproed on:https://github.com/dotnet/aspnetcore/blob/3ce1231d5c3900b7629c283d47c124dafb26903e/src/Components/Server/src/Circuits/ServerComponentDeserializer.cs#L229

```C#
private static readonly Action<ILogger, string, Exception> _invalidMarkerType =
                LoggerMessage.Define<string>(
                    LogLevel.Debug,
                    new EventId(4, "InvalidMarkerType"),
                    "Invalid component marker type '{}'.");
```

Stack trace:

```C#
System.IndexOutOfRangeException: Index was outside the bounds of the array.
at Microsoft.NetCore.Analyzers.Runtime.LoggerMessageDefineAnalyzer.AnalyzeFormatArgument(OperationAnalysisContext context, IOperation formatExpression, Int32 paramsCount, Boolean argsIsArray, Boolean usingLoggerExtensionsTypes, IMethodSymbol methodSymbol)
at Microsoft.NetCore.Analyzers.Runtime.LoggerMessageDefineAnalyzer.AnalyzeInvocation(OperationAnalysisContext context, INamedTypeSymbol loggerType, INamedTypeSymbol loggerExtensionsType, INamedTypeSymbol loggerMessageType)
at Microsoft.NetCore.Analyzers.Runtime.LoggerMessageDefineAnalyzer.<>c__DisplayClass27_0.<Initialize>b__1(OperationAnalysisContext context)
at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.<>c.<ExecuteOperationAction>b__63_0(ValueTuple`2 data)
at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock[TArg](DiagnosticAnalyzer analyzer, Action`1 analyze, TArg argument, Nullable`1 info)
```

Fixing the parameter resolved the issue.

**Comment by pranavkm** [comment]
FYI @maryamariyan

**Comment by maryamariyan** [comment]
ok looking

**Comment by maryamariyan** [comment]
fixed via #5384
